skops/io/tests/test_persist.py

```diff
@@ -67,6 +67,47 @@
 # Python 3.8)
 ATOL = 1e-6 if sys.platform == "darwin" else 1e-7
 
+# TODO: remove when these are added to trusted types
+SKLEARN_EXCLUDED_TYPES = (
+    "sklearn._loss._loss.CyHalfBinomialLoss",
+    "sklearn._loss._loss.CyHalfGammaLoss",
+    "sklearn._loss._loss.CyHalfPoissonLoss",
+    "sklearn._loss._loss.CyHalfSquaredError",
+    "sklearn._loss._loss.CyHalfTweedieLossIdentity",
+    "sklearn._loss.link.IdentityLink",
+    "sklearn._loss.link.Interval",
+    "sklearn._loss.link.LogLink",
+    "sklearn._loss.link.LogitLink",
+    "sklearn._loss.loss.HalfBinomialLoss",
+    "sklearn._loss.loss.HalfGammaLoss",
+    "sklearn._loss.loss.HalfPoissonLoss",
+    "sklearn._loss.loss.HalfSquaredError",
+    "sklearn._loss.loss.HalfTweedieLossIdentity",
+    "sklearn.calibration._CalibratedClassifier",
+    "sklearn.calibration._SigmoidCalibration",
+    "sklearn.cluster._bisect_k_means._BisectingTree",
+    "sklearn.cluster._kmeans._kmeans_single_lloyd",
+    "sklearn.covariance._graph_lasso._DictWithDeprecatedKeys",
+    "sklearn.ensemble._gb_losses.BinomialDeviance",
+    "sklearn.ensemble._gb_losses.LeastSquaresError",
+    "sklearn.ensemble._hist_gradient_boosting.binning._BinMapper",
+    "sklearn.ensemble._hist_gradient_boosting.predictor.TreePredictor",
+    "sklearn.feature_selection._univariate_selection.f_classif",
+    "sklearn.gaussian_process._gpc._BinaryGaussianProcessClassifierLaplace",
+    "sklearn.gaussian_process.kernels.ConstantKernel",
+    "sklearn.gaussian_process.kernels.Product",
+    "sklearn.gaussian_process.kernels.RBF",
+    "sklearn.impute._iterative._ImputerTriplet",
+    "sklearn.metrics._dist_metrics.EuclideanDistance",
+    "sklearn.metrics._scorer._passthrough_scorer",
+    "sklearn.model_selection._split.StratifiedKFold",
+    "sklearn.multiclass._ConstantPredictor",
+    "sklearn.neighbors._ball_tree.BallTree",
+    "sklearn.neighbors._kd_tree.KDTree",
```

Comment:
Interestingly `sklearn.neighbors._ball_tree.BallTree` and `sklearn.neighbors._kd_tree.KDTree` are not present in the list returned by `all_estimators`